app/src/main/java/org/simple/clinic/mobius/MobiusDelegate.kt

```diff
@@ -21,7 +22,7 @@ import org.simple.clinic.util.unsafeLazy
 class MobiusDelegate<M : Parcelable, E, F>(
     private val events: Observable<E>,
     private val defaultModel: M,
-    private val init: Init<M, F>,
+    private val init: Init<M, F>?,
```

Comment:
Instead of making this nullable, what do you think of making it an optional parameter and setting `Init { first(defaultModel) }` as a default value?

Reply:
I don't see any significant gain in doing that.